### ISSUE TITLE
csproj change configured encoding on add file

### ISSUE THREAD

**Issue by sirber** [jRichardeau/vscode-vsproj#4]
![image](https://user-images.githubusercontent.com/3911428/114042744-b1586580-9853-11eb-959e-cfc297bfd470.png)

**Comment by jRichardeau** [comment]
Hi Sirber,

Yes this is the default configuration but you can change it with the `vsproj.encoding` option, it's in the extension documentation.

**Comment by jRichardeau** [comment]
try `utf8 ` or `utf-8`, don't remember

**Comment by sirber** [comment]
why change it if not configured? why not get it from the project file xml?

**Comment by jRichardeau** [comment]
Yep you're right, didn't have the time to do it, feel free to create a pull request :)

**Comment by sirber** [comment]
I can't promise, but it the meantime, this issue should stay open.

**Comment by sirber** [comment]
utf-8 saves the file whitout "bom"

**Comment by jRichardeau** [comment]
Oh yes it's saving with utf-8 without bom, could be an improvement.
